tests/test_pluggable.py

```diff
@@ -141,6 +141,7 @@ def test_from_info_full_options(self, mock_init):
             subject_token_type=SUBJECT_TOKEN_TYPE,
             token_url=TOKEN_URL,
             service_account_impersonation_url=SERVICE_ACCOUNT_IMPERSONATION_URL,
+            service_account_impersonation_info=None,
```

Comment:
Add a test where the service_account_impersonation_options is there

Reply:
Done.  Also in pluggable and identity_pool